source/features/bypass-checks.tsx

```diff
@@ -15,8 +15,14 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 		return;
 	}
 
-	const directLink = await api.v3(`repos/${getRepoURL()}/check-runs/${runId}`);
-	detailsLink.href = directLink.details_url;
+	const {owner, name: repo} = getRepositoryInfo();
+	const check_run_id = Number.parseInt(runId, 10);
+	if (!owner || !repo || !check_run_id) {
```

Comment:
Why the check?

Reply:
The `check_run_id` one?
Just for good measure if for some reason the `runId` could not be parsed into an int.

I guess it would also be possible to pass the `runId` directly as a `Number`.

_(I'm also not an expert in Typescript, so there might be better ways in general :smile:)_